ts/components/conversation/header/ConversationHeader.tsx

```diff
@@ -131,9 +134,24 @@ function RecreateGroupButton() {
   return (
     <RecreateGroupContainer>
       <SessionButton
-        buttonColor={SessionButtonColor.Primary}
+        buttonType={SessionButtonType.Outline}
         margin="var(--margins-sm)"
-        onClick={() => {
+        onClick={async () => {
+          try {
+            for (let index = 0; index < members.length; index++) {
```

Comment:
I would prefer `/* eslint-disable no-await-in-loop */` to be on this block please and not for the entire file.

Reply:
If we do this in showRecreateGroupModal as well it feels redundant?